hack/tidy-check.sh

```diff
@@ -64,14 +66,7 @@ function tidy {
   mv "$SUM_FILE" "$TMP_SUM_FILE"
 
   if [ -n "$GO" ] && [[ "$($GO version|awk '{print $3}')" == $TARGET_GO_VERSION_PATTERN ]]; then
```

Comment:
so what happens if this condition is not true?

Reply:
I thought we only allow to run under go1.13.  Will update this part.

Reply:
Done.